### PR TITLE
#2285-Add-example-scripts-hyperloglog - Added hyperloglog examples to…

### PR DESCRIPTION
**Pull request by ade1705**
Added example scripts for the hyperloglog as requested - #2285 

created a hyperloglog.js file

### Checklist

- [ ] Does `npm test` pass with this change (including linting)?
- [ ] Is the new or changed code fully tested?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?

### CODE REVIEW

**Comment by SuzeShardlow** [comment]
Closes #2285 when merged.

File: examples/hyperloglog.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+//Example to log traffic data at intersections for the city of San Francisco.
+//Log license plates of each car scanned at each intersection and add to the intersections Hyperloglog
```
Comment (simonprickett) [inline examples/hyperloglog.js:2]:
```suggestion
// Log license plates of each car scanned at each intersection and add to the intersections Hyperloglog.
```

**Review by simonprickett — CHANGES_REQUESTED** [review]
Thank you - looking good, the code works well thanks.  I've left a couple of suggestions, please ensure all comments have a space between the `//` and the text, and that sentences end with a full stop / period... so:

```
// this is a comment.
```

rather than:

```
//this is a comment
```

Thank you!

**Comment by ade1705** [comment]
@simonprickett Thanks for the feedback, Changes made

**Comment by SuzeShardlow** [comment]
Thanks @ade1705, we'll review this!  In the meantime, if you're a member of our Discord server (https://discord.gg/redis), please let us know your handle.  Thanks!

**Comment by ade1705** [comment]
> Thanks @ade1705, we'll review this! In the meantime, if you're a member of our Discord server (https://discord.gg/redis), please let us know your handle. Thanks!

Yea, I am a member of the discord, my handle `@TrussDamola`

**Comment by SuzeShardlow** [comment]
Closes #2285.  Thanks @ade1705!  I'll give you the `Hacktoberfest 2022` role on Discord :)

**Pull request merged**